docs/getting-started.md

```diff
@@ -57,16 +57,34 @@ $ kubectl create -f helm/generated_examples/gce.yaml
 
 #### Option 2: GKE
 
-GKE clusters will automatically format and mount the
-requested Local SSDs. Please see
+GKE clusters have raw block local NVMe SSDs attached. Please see
 [GKE
-documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/local-ssd)
+documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/local-ssd#raw-block)
 for instructions for how to create a cluster with Local SSDs.
 
-Then skip to [step 4](#step-4-create-local-persistent-volume-claim).
-
-**Note:** The raw block feature is only supported on GKE Kubernetes alpha clusters.
+After you create the cluster, use the examples/gke-daemonset-raid-disks.yaml to 
+RAID and format your local SSD disks. This DaemonSet will set RAID0 array on all 
+local SSD disks and format the device to an ext4 filesystem.
 
+```
+$ kubectl create -f examples/gke-daemonset-raid-disks.yaml
+```
+Use helm/generated_examples/gke-nvme-ssd-block-raid.yaml to setup provisioner
```

Comment:
nit: these could be markdown links, in the preview they're still text

Reply:
I saw that you can do this with https://github.blog/2013-01-31-relative-links-in-markup-files/

Reply:
Also I think you could squash your changes into 1 change

Reply:
Done